frontend/src/store/reducers/index.js

```diff
@@ -14,5 +15,6 @@ export default combineReducers({
   currentSurvey: currentSurveyReducer,
   general: generalReducer,
   surveys: surveysReducer,
-  results: resultsReducer
+  results: resultsReducer,
+  answers
```

Comment:
Let's keep it similar:
1. Use `answer: answerReducer` 
**or**
2. Add aliases for others reducers

Reply:
done